compiler/rustc_ty_utils/src/needs_drop.rs

```diff
@@ -145,10 +141,8 @@ where
                             Ok(tys) => tys,
                         };
                         for required_ty in tys {
-                            let subst_ty = tcx.normalize_erasing_regions(
-                                self.param_env,
-                                required_ty.subst(tcx, substs),
-                            );
+                            let subst_ty =
```

Comment:
Note: I probably would have renamed this variable. I think the reason it was named `subst_ty` is because it, prior to this PR, had the `substs` applied to it. But that is no longer the case here, if I understand correctly.